browser/scripts/loaders/multiObjectLoader.js

```diff
@@ -7,6 +7,8 @@ function MultiObjectLoader(manager) {
 	this.objectLoader = new THREE.ObjectLoader(manager)
```

Comment:
no, ObjectLoader is included in the main three.js file. This is removing a duplicate